src/pages/user/growth.jsx

```
@@ -89,241 +101,71 @@ const growth = () => {
+                  <CardContent className="text-base pb-3 text-justify">
+                    <p>{food.content}</p>
+                  </CardContent>
+                  {food.badgeText && (
+                    <CardFooter className="pb-0">
+                      <Badge variant="secondary" className="bg-light-badge-green text-light-success-green">{food.badgeText}</Badge>
+                    </CardFooter>
+                  )}
+                </div>
+              </Card>
+            </div>
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)